docs/labwc-config.5.scd

```diff
@@ -330,6 +330,50 @@ The rest of this man page describes configuration options.
 	any motion events while a keyboard is typing, and for a short while
 	after as well.
 
+## WINDOW RULES
+
+Two types of window rules are supported, actions and properties. They are
+defined as shown below. Property values can be *yes*, *no* or *unset*.
```

Comment:
I am not sure we need to expose the `unset` argument to the user.
We need it internally to differentiate between a user supplying `no` vs not setting the property at all though.

Reply:
The reason for 'unset' is that if a property has already been set to yes/no using for example `identifier="*"`, we might then want to unset the property for a particular window in order for it to fall back onto default values (or in the example server decoration, the two related protocol implementations).

Reply:
What do you think about renaming it to `default` in that case?
`unspec` feels a bit "codish" to me for a user facing config file.

Reply:
Done